mzLib/MzLibUtil/DoubleRange.cs

```diff
@@ -135,5 +138,15 @@ public bool Contains(double item)
         {
             return CompareTo(item).Equals(0);
         }
+
+        /// <summary>
+        /// Determines if a majority of the items are within the range of values
```

Comment:
items?

Reply:
This is the convention used with the contains method above